package.json

```diff
@@ -26,6 +26,8 @@
     "start": "nodemon --ignore test/"
   },
   "dependencies": {
+    "cryptiles": "^4.1.3",
+    "growl": "^1.10.5",
```

Comment:
I'm not familiar with these 2 dependencies. What are you using them for here?

Reply:
They were giving some security alerts on github. So updated their version and saved them in package.json

Reply:
I wonder if there is another way to force it to use a higher version. I'm not sure you'd want to do this in a real application, because this is saying that growl and cryptiles are direct dependencies of your app, and they are not.

Reply:
Agreed. I think they are dependencies of Hapi and mocha libraries